docs/maintenance.md

```diff
@@ -0,0 +1,20 @@
+# OSPS Baseline Maintenance Process
+
+* Normal text fixes to the criteria will be accepted via pull request and reviewed by the baseline project maintainers.
+Allowed changes are corrections to spelling/typos, grammar corrections, or enhancements to the supplementary text supporting the criteria including - Objective, Implementation, Control Mappings, and Scorecard/Insights values.
+At least two project maintainers must review and approve these changes.
+* Substantive changes to Criteria, including changes to text that alters the originally stated meaning, new Criteria proposals, or removal of Criteria will be documented in GitHub PR(s) and reviewed regularly by the Baseline project maintainers.
+These changes may reflect changes to global cybersecurity regulations and frameworks or changes in norms around application/project security practices.
+Any such substantive changes must be approved by a majority of the project's maintainers.
+* As appropriate, but at least annually, the Baseline project maintainers will publish a new version of the Baseline.
+Previous versions of the Baseline will remain avialable, but are not maintained.
+* Any changes to the Baseline will be reflected within the Compliance Matrix, with new requirements flagged where the Baseline Criteria are appropriate.
+* Downstream stakeholders will be notified via the project's mailing list on the changes and updates.
+
+## Identifiers
+
+* Identifiers for retired criteria will not be used.
```

Comment:
```suggestion
* Identifiers for retired criteria MUST NOT be reused.  A list of these retired identifiers will be maintained in (file that contains at least OSPS-AC-06 and OSPS-BR-07)
```

Reply:
I like the idea of keeping them in the yaml with a `retired: <BOOLEAN>` entry, which is what I think puerco's comment hinted at.

Reply:
If you want to keep a single file across versions (I think you're gonna regret it, but maybe I'll learn something), you might want `retired: $VERSION` rather than a boolean.

Reply:
Ah, you'll just check in the file as of a certain point, so then you use text-diff to find the changes over time.  I think that will probably be workable with tools like `blame`.

Reply:
Good point on `retired: $VERSION`. `git blame` would give the same info, but this would be friendlier.